### PR TITLE
[DEV-4321] New API Contracts to support Federal Account page

### PR DESCRIPTION
**Pull request by tony-sappe**
**Description:**
API Contracts to lay the groundwork for migrating Federal Account page from v1 endpoints to v2

**Technical details:**
N/A

**Requirements for PR merge:**

1. [x] Unit & integration tests updated (N/A)
2. [x] API documentation updated
3. [ ] Necessary PR reviewers:
    - [x] Backend
    - [x] Frontend
    - [ ] Domain Expert
4. [x] Matview impact assessment completed (N/A)
5. [x] Frontend impact assessment completed
6. [x] Data validation completed (N/A)
7. [x] Appropriate Operations ticket(s) created (N/A)
8. [x] Jira Ticket [DEV-4321](https://federal-spending-transparency.atlassian.net/browse/DEV-4321):
    - [x] Link to this Pull-Request
    - [x] Performance evaluation of affected (N/A)
    - [x] Before / After data comparison (N/A)

**Area for explaining above N/A when needed:**
```
Only new API Contracts in this PR
```

### CODE REVIEW

**Comment by jonhill13** [comment]
From my knowledge base and comparing current search results and federal account pages it looks. 👍

File: usaspending_api/api_contracts/contracts/v2/federal_accounts/federal_account_id/available_program_activity.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# List of Available Program Activities [/api/v2/federal_accounts/{federal_account_id}/available_program_activity]
```
Comment (ebdabbs) [inline usaspending_api/api_contracts/contracts/v2/federal_accounts/federal_account_id/available_program_activity.md:4]: Should we make this plural (`available_program_activities`) for consistency with `available_object_classes`?

File: usaspending_api/api_contracts/contracts/v2/federal_accounts/federal_account_id/available_program_activity.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+This endpoint supports the Federal Account page and allow for listing all Program Activities with activity by the Federal Account.
```
Comment (ebdabbs) [inline usaspending_api/api_contracts/contracts/v2/federal_accounts/federal_account_id/available_program_activity.md:6]: `allow for` --> `allows for` (similar below)

File: usaspending_api/api_contracts/contracts/v2/federal_accounts/federal_account_id/spending_over_time.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
+FORMAT: 1A
+HOST: https://api.usaspending.gov
+
+# List of Available Program Activities [/api/v2/federal_accounts/{federal_account_id}/spending_over_time]
```
Comment (ebdabbs) [inline usaspending_api/api_contracts/contracts/v2/federal_accounts/federal_account_id/spending_over_time.md:4]: Looks like this was copy-pasted from the Program Activity endpoing. Suggested heading/description - `Account Spending by Time Period` / `This endpoint supports the Federal Account page and allows for listing spending by the Federal Account within certain periods of time.`

**Pull request merged**